src/main/java/org/datadog/jenkins/plugins/datadog/traces/DatadogTraceBuildLogic.java

```diff
@@ -99,6 +100,11 @@ public void finishBuildTrace(final BuildData buildData, final Run<?,?> run) {
         buildSpan.setTag(prefix + CITags._NUMBER, buildData.getBuildNumber(""));
         buildSpan.setTag(prefix + CITags._URL, buildData.getBuildUrl(""));
 
+        final TimeInQueueAction timeInQueueAction = run.getAction(TimeInQueueAction.class);
+        if(timeInQueueAction != null) {
+            buildSpan.setTag(prefix + CITags._QUEUE_TIME, timeInQueueAction.getSecondsInQueue());
```

Comment:
Not familiar with how the intake will work, but tagging with a float is a dangerous thing for metrics leading to context explosion very fast.
Are we sure the backend can handle float values as tags in an APM context?

Reply:
I'm confused. The tag is based on a `long` value. In this case, the number of seconds calculated as `long` in https://github.com/jenkinsci/datadog-plugin/blob/master/src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogBuildListener.java#L108

Reply:
Sorry I've used "float" for "number" in that context. The point being that the value of the tag will be different for each span

Reply:
Ah!, It's totally supported by the backend. No worries. It's based on the Opentracing standard.

![image](https://user-images.githubusercontent.com/29516565/92928770-df053980-f43f-11ea-906c-f2fce374b94e.png)